src/emailservice/email_server.py

```diff
@@ -143,7 +144,24 @@ def start(dummy_mode):
   except KeyboardInterrupt:
     server.stop(0)
 
+def InitProfiler():
+  for retry in range(3):
```

Comment:
use range (1,4) here please, it's only retrying for 20 seconds because retry=0

Reply:
done